store/sql_spinmint_store_test.go

```diff
@@ -0,0 +1,67 @@
+package store
+
+import (
+	"testing"
+
+	"github.com/mattermost/mattermost-mattermod/model"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSQLSpinmintStore(t *testing.T) {
+	ss := getTestSQLStore(t)
+
+	sms := NewSQLSpinmintStore(ss)
+	err := ss.master.CreateTablesIfNotExists()
+	require.NoError(t, err)
+
+	sm := &model.Spinmint{
+		RepoName: "repo-name",
+		Number:   123,
+	}
+
+	t.Run("no rows on Get", func(t *testing.T) {
+		npr, err := sms.Get(123, "repo-name")
+		require.NoError(t, err)
+		assert.Nil(t, npr)
+	})
+
+	t.Run("happy path on Save", func(t *testing.T) {
```

Comment:
Let's also test an upsert scenario by saving a spinmint with a new value and confirm that it gets updated.